content/Voice Over - Fernanda Toledo.md

```diff
@@ -0,0 +1,63 @@
+# How to make your app accessible using Voice Over
+
+![](https://github.com/rootstrap/blog/blob/voice-over/content/images/glasses.jpg)
+
+In a world where everything is relying more and more on technology, more tools are being provided for us developers to make our apps more accessible to everyone. Having limited sight should not be an impediment to use any app. As developers or designers, we need to raise awareness of all of those we're leaving behind and take a little time to make sure that our apps are accessible to everyone. Everyone wins, you'll broaden your market and increase your exposure, which will always lead to a positive impact on your profits. And the most important, by including people with disabilities you'll make a huge difference in their lives!
+
+So, what makes an app accessible? According to the [Web Accessible Initiative](https://www.w3.org/WAI/fundamentals/accessibility-intro/), it means that the app has been designed so that people with disabilities are capable of using it, meaning they can not only perceive and understand it but also navigate and interact with the app. With that said, how can we make our apps accessible? There is a broad set of tools to help users with disabilities, we need to take action and remove the accessibility barriers by providing alternative solutions with the help of these tools. Specifically, in this article we'll talk about how to use, test, and improve accessibility using Voice Over.
+
+### What is Voice Over?
+
+For users that have a very low or no sight, there's Voice Over, a gesture-based screen reader that allows users to know what's on the screen by playing audible descriptions. The user can navigate through the screen by swiping and they can trigger different actions depending on a given [set of gestures](https://support.apple.com/guide/iphone/learn-voiceover-gestures-iph3e2e2281/13.0/ios/13.0).
+
+### Try it on one of your apps!
+
+Open one of the apps you've developed and turn on Voice Over, read the entire content of the screen by swiping up using two fingers and voila, by default Voice Over will read some things, so that's a good head start, some buttons can be tapped by using double tap and some labels may be read correctly, but I can guarantee you that if accessibility was not taken into account in the design and development stages, then there will be issues. There might be things that are not being read in the correct order, things that are not being read at all, and some elements might not be grouped correctly because of a wrong view hierarchy. Some common mistakes can be that an uppercase text is by default read one letter at a time (OMG, for instance, should be read Oh my god, but instead the 3 letters will be read), and in case you have checkboxes probably the state of the checkbox is not being announced. If your app has those issues, users with disabilities won't be able to use it, because those things make it impossible for them to move forward.
```

Comment:
```suggestion
Open one of the apps you've developed and turn on VoiceOver, read the entire content of the screen by swiping up using two fingers and voila, by default VoiceOver will read some things, so that's a good head start, some buttons can be tapped by using double tap and some labels may be read correctly, but I can guarantee you that if accessibility was not taken into account in the design and development stages, then there will be issues. There might be things that are not being read in the correct order, things that are not being read at all, and some elements might not be grouped correctly because of a wrong view hierarchy. Some common mistakes can be that an uppercase text is by default read one letter at a time (OMG, for instance, should be read Oh my god, but instead the 3 letters will be read), and in case you have checkboxes probably the state of the checkbox is not being announced. If your app has those issues, users with disabilities won't be able to properly use it, because those things make it impossible for them to move forward.
```